src/Section1/style.js

```diff
@@ -1,45 +1,94 @@
-
-import styled from 'styled-components';
-import { ReactComponent as Img } from '../programming 2.svg';
-
-
+import styled from "styled-components";
+import { ReactComponent as Img } from "../programming 2.svg";
 
 export const Container = styled.section`
-
-width: 100%;
-height: 608px;
-background: #FFD66C;
+  position: relative;
+  width: 100%;
+  height: 597px;
+  background: #ffd66c;
+
+  @media (max-width: 768px) {
+    width: 100%;
+    height: 603px;
+  }
+
+  @media (max-width: 376px) {
+    width: 100%;
+    height: 439px;
+  }
 `;
 
 export const Title = styled.h1`
-
-width: 515px;
-height: 167px;
-
-margin-bottom: 302px;
-margin-left: 44px;
-padding-top:128px;
-
-
-font-style: normal;
-font-weight: bold;
-font-size: 74px;
-line-height: 87px;
-
-color: #FFFFFF;
+  width: 515px;
+  height: 167px;
+
+  padding-bottom: 302px;
+  padding-left: 44px;
+  padding-top: 128px;
+
+  font-style: normal;
+  font-weight: bold;
+  font-size: 74px;
+  line-height: 87px;
+
+  color: #ffffff;
+
+  @media (max-width: 768px) {
+    width: 498px;
+    height: 141px;
+
+    padding-bottom: 308px;
+    padding-left: 47px;
+    padding-top: 154px;
+
+    font-style: normal;
+    font-weight: 700;
+    font-size: 65px;
+    line-height: 76px;
+  }
+
+  @media (max-width: 376px) {
+    width: 268px;
+    height: 109px;
+
+    padding-bottom: 198px;
+    padding-left: 40px;
+    padding-top: 132px;
+
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 47px;
+  }
 `;
 
 export const Image = styled(Img)`
-position: absolute;
-width: 544.14px;
-height: 395px;
-left: 434px;
-top: 289px;
-
+  position: absolute;
+  width: 544.14px;
+  height: 395px;
```

Comment:
To simplify your life, whenever you change the height or width of an SVG you can say the other property is "auto" so it will just adjust proportionally to the initial size.

in here you could just check the width of the SVG and set `height: auto;`, then you can just erase the heights in your media queries and variate only the width.